### PR TITLE
2.0.0-alpha reorganisation

### PR DESCRIPTION
**Pull request by jchannon**

### CODE REVIEW

File: src/Nancy.Serialization.ServiceStack/Nancy.Serialization.ServiceStack.xproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0.24720</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DNX\Microsoft.DNX.Props" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>fd7096a0-e81d-4a62-b0c6-5049f2e42898</ProjectGuid>
+    <RootNamespace>Nancy.Serialization.ServiceStack</RootNamespace>
+    <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
```
Comment (khellang) [inline src/Nancy.Serialization.ServiceStack/Nancy.Serialization.ServiceStack.xproj]: Is this path correct? I think it's a level too high?

File: src/Nancy.Serialization.ServiceStack/project.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿{
+    "version": "1.0.0-*",
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Should be `2.0.0-*`

Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Actually, the rest of the properties are wrong as well

File: src/Nancy.Serialization.ServiceStack.MSBuild/Nancy.Serialization.ServiceStack.csproj [hunk 3]
```diff
@@ -47,15 +47,15 @@
...
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="..\..\..\dependencies\Nancy\SharedAssemblyInfo.cs">
+    <Compile Include="..\..\dependencies\Nancy\SharedAssemblyInfo.cs">
       <Link>Properties\SharedAssemblyInfo.cs</Link>
     </Compile>
-    <Compile Include="Helpers.cs" />
-    <Compile Include="ServiceStackBodyDeserializer.cs" />
-    <Compile Include="ServiceStackJsonSerializer.cs" />
+    <Compile Include="..\Nancy.Serialization.ServiceStack\Helpers.cs" />
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack.MSBuild/Nancy.Serialization.ServiceStack.csproj]: These should be links

File: src/Nancy.Serialization.ServiceStack/project.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+﻿{
+    "version": "2.0.0-*",
+
+    "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Wrong description, check the `nuspec` file

File: src/Nancy.Serialization.ServiceStack/project.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+﻿{
+    "version": "2.0.0-*",
+
+    "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",
+    "authors": [ "Andreas Håkansson, Steven Robbins and contributors" ],
+    "tags": [ "Nancy" ],
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Wrong tags, check the `nuspec` file

File: src/Nancy.Serialization.ServiceStack/project.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "licenseUrl": "https://github.com/NancyFx/Nancy/blob/master/license.txt",
+    "iconUrl": "http://nancyfx.org/nancy-nuget.png",
+
+    "dependencies": {
+        "Nancy": { "target": "project" },
+        "ServiceStack.Text": "3.9.71"
+    },
+
+    "frameworks": {
+
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Remove blank line

File: src/Nancy.Serialization.ServiceStack/project.json [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+        "net451": {
+
```
Comment (thecodejunkie) [inline src/Nancy.Serialization.ServiceStack/project.json]: Remove blank line and move the closing brace to line 18

**Comment by thecodejunkie** [comment]
Could you add the solution folders to the `*.sln` as well please

**Pull request merged**